integration_tests/pages/page.ts

```diff
@@ -5,14 +5,38 @@ export default abstract class Page {
     return new constructor()
   }
 
-  constructor(private readonly title: string) {
+  constructor(
+    private readonly title: string,
+    private readonly options: { axeTest?: boolean } = {
+      axeTest: true,
+    },
+  ) {
     this.checkOnPage()
+    this.checkCsfrTokenForFormBasedPages()
```

Comment:
After checking we are on the correct page (based on the h1 element), we check that pages with forms have a csrf token (so we cant forget to do it 👍 ); plus if the axe option is set it runs the axe tests 👍